### DISCUSSION TITLE
Add the possibility of translations by default

### DISCUSSION THREAD

**Discussion by Voldemorten** [nuotsu/sanitypress-2023#25] [Ideas]
Hi! 

I think it would be valuable to add default translations. Maybe just start with one language in english, but then the user will have an easier job including more locales themselves. 

Other than that; great template!

**Comment by mitchuman** [comment]
@Voldemorten Thanks for the suggestion and kind comment!

Localization/i18n is definitely something I'd eventually want to incorporate into the template. As I have not worked with any Sanity plugins or have tried to implement myself, I'd ideally want to try a solution first before adding to the template.

With that, my current stance is: _Localization is left to the adopter to implement, as there are a number of community plugins available_ (at least for now)

**Comment by nuotsu** [comment]
I'm thrilled to announce i18n is now supported in SanityPress v7.0.0. See [SanityPress Pro](https://sanitypress.dev/pricing) for details.
